From 9f8db44c53e25b7fe54816616f589f9897a63dd5 Mon Sep 17 00:00:00 2001
From: ryang <decatf@gmail.com>
Date: Wed, 1 Feb 2017 14:34:31 -0500
Subject: [PATCH 1/2] chromium src build fixes for non-NEON

---
 third_party/libwebp/BUILD.gn | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/third_party/libwebp/BUILD.gn b/third_party/libwebp/BUILD.gn
index 470bd29..871f4c7 100644
--- a/third_party/libwebp/BUILD.gn
+++ b/third_party/libwebp/BUILD.gn
@@ -13,8 +13,7 @@ set_opt_level =
     !is_debug && is_posix && (current_cpu == "arm" || current_cpu == "arm64")
 
 use_dsp_neon =
-    current_cpu == "arm64" || (current_cpu == "arm" && arm_version >= 7 &&
-                               (arm_use_neon || arm_optionally_use_neon))
+    current_cpu == "arm64" || (current_cpu == "arm" && arm_version >= 7)
 
 static_library("libwebp_dec") {
   sources = [
-- 
2.7.4

